tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = test
+envlist = test{py27,py33,py34}
```

Comment:
tox magically parses the ``envlist`` to determine which interpreter to use. Unfortunately it's quite specific about the format of the name. The above actually creates three environments all using the default interpreter.

Any of the following would work as you intended:
* ``envlist = {py27,py33,py34}``
* ``envlist = {py27,py33,py34}-test``
* ``envlist = foobar-{py27,py33,py34}``

I prefer the first one I think